src/atomic/main.cpp

```diff
@@ -455,6 +462,12 @@ int main(int argc, char **argv) {
   if(Zl!=0 || Zr !=0)
     printf("Done in %.6f\n",tnuc.get());
 
+  // Confinement potential
+  if(conf_N)
+    printf("Computing confinement potential\n");
+  arma::mat Vconf=basis.confinement(conf_N, conf_R, iconf);
```

Comment:
Message is not consistent with code.
```suggestion
arma::mat Vconf(basis.Nbf(),basis.Nbf(),arma::fill::zeros);
if(conf_N) {
    printf("Computing confinement potential\n");
    Vconf=basis.confinement(conf_N, conf_R, iconf);
}
```